code_annotations/__init__.py

```diff
@@ -4,4 +4,4 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = '0.3.2'
+__version__ = '0.3.3'
```

Comment:
Question -- Should this be a fix version update? To me changing the underlying django version would be a bigger change set, no? I wouldn't be surprised if this was a major change version.

Reply:
I think that just bumping the patch version here is appropriate.  A major version change would mean that we are making a breaking change to an api in this package--we're actually not changing anything as far as users of code-annotations are concerned.  A minor version change would be if we're adding functionality (which we are not).  We're just making ourselves compatible with a later version of one of our dependencies.